### DISCUSSION TITLE
Builder doesn’t work

### DISCUSSION THREAD

**Discussion by niccuc** [Budibase/budibase#9549] [Help] [answered]
Hi, today I have problems with budibase builder. The website doesn’t work: black screen. Can you help me?

**Comment by andz-bb** [comment, marked as answer]
hey @niccuc, sorry to hear you are having problems with the builder today. Are you using a cloud or a self-hosted setup? Does the browser's console show any errors when you open the developer tools (F12 on Chrome)? Thanks

**Comment by niccuc** [comment]
No errors: only empty screen. In iOS and iPadOS the error there is with every browser. On Android it works. On Mac with chrome it works, with safari doesn’t work. Maybe an “Apple” error?

**Reply by andz-bb** [reply]
@niccuc thanks for the info! I'm seeing the same on my cloud tenant with macOS and iPadOS Safari. Sorry for the inconvenience, we're looking into it now and will hopefully have a fix released very soon!

**Reply by andz-bb** [reply]
@niccuc A fix for the builder in Safari was released last night. Hope this helps!

<img width="1380" alt="Screenshot 2023-02-07 at 09 59 47" src="https://user-images.githubusercontent.com/110921612/217213285-307829ca-6811-49bf-90a8-cd27aa3fd37d.png">

Regarding your message below, I have sent an email directly to the address associated with your Budibase cloud account

**Comment by niccuc** [comment]
Hi Andrew, in this regard, I wanted to ask you something.  My son, who is
studying computer programming at university, has designed, thanks to
budibase, an application for us to have fun with our friends, from 7 to 11
February, inserting the votes for the singers of a famous song festival
(Sanremo) here in Italy.  Basically, watching the singers on TV, we enter
the votes and we look at our personal rankings in real time (it seems silly
but every year we have fun like this).  After days of effort and having
prepared everything, we have only now realized that, to have our friends
test the application and fix the various bugs, we have practically used up
almost all the daily passes (we only now realized that there were limited
and we wasted them to make our friends vote at random while waiting for the
festival that starts tomorrow).  I swear we're desperate because it all
starts tomorrow and there's no time to think of another solution.  there
wouldn't be a way to reset the passes (40 aren't many for us), but at least
we could let 7/8 people use it for the next 5 days?  I know it sounds like
an absurd request, but we were really disappointed.  Thank you very much
and sorry if somehow with this request I can be clumsy or rude.  Thank you.

Il giorno lun 6 feb 2023 alle 17:08 Andrew Thompson <
***@***.***> ha scritto:

**Reply by andz-bb** [reply]
hey @niccuc, what is your Budibase cloud tenant ID? that'll be the name in your URL before `.budibase.app`

**Comment by niccuc** [comment]
It’s leocucu

Il giorno lun 6 feb 2023 alle 21:34 Andrew Thompson <
***@***.***> ha scritto:

**Comment by niccuc** [comment]
Thank you so much. As my son wake up he will read his emails ☺️

Il giorno mar 7 feb 2023 alle 11:02 Andrew Thompson <
***@***.***> ha scritto:
